### ISSUE TITLE
Update use of request_futures: background_callback is being deprecated

### ISSUE THREAD

**Issue by sarasafavi** [planetlabs/planet-client-python#171]
```
FuturesSession - WARNING - `background_callback` is deprecated and will be removed in 1.0, use `hooks` instead
```

**Comment by bosth** [comment]
Is there a straight-forward way to disable this when writing code with the client?

**Comment by bosth** [comment]
#202 will remove this warning by using a specific version of `requests-futures`.

**Comment by sgillies** [comment]
Resolved by #202.
